packages/hub/process-controllers/hub-bot-controller.ts

```diff
@@ -0,0 +1,47 @@
+import { runInitializers, createRegistry } from '../main';
+import HubBot from '../services/discord-bots/hub-bot';
+import * as Sentry from '@sentry/node';
+import { Registry, Container } from '@cardstack/di';
+import { botLog } from '../utils/logger';
+
+export class HubBotController {
```

Comment:
So the way the hub bot and worker controllers do thier DI is actually inverted and we have tickets to fix them. that being said we should stop repeating this pattern. The idea is that the controller should not create the container, rather the controller lives in the container. The correct pattern to follow is the HubServer. Where it is dependency injected into the container (rather than creating the container). You can refer to tickets #CS-2707 and #CS-2606